MoltenVK/MoltenVK/Utility/MVKEnvironment.h

```diff
@@ -357,5 +357,12 @@ void mvkSetConfig(MVKConfiguration& dstMVKConfig, const MVKConfiguration& srcMVK
 #	define MVK_CONFIG_USE_METAL_PRIVATE_API MVK_USE_METAL_PRIVATE_API
 #endif
 
+/**
+ * If set, MVK will dump spirv input, translated msl, and pipelines into the given directory.
+ */
```

Comment:
Align with other comments.

```suggestion
/** If set, MVK will dump spirv input, translated msl, and pipelines into the given directory. */
```